Automattic/wp-calypso
# Launchpad: Change hover style for "Choose a Plan task"

#### Summary
There's been a [recent PR](https://github.com/Automattic/wp-calypso/pull/67763) for enabling going back to the **Choose a Plan** task when it has been completed. Since the task has both a badge and a chevron icon there has been some discussion regarding styling. This is how it looks currently:

https://user-images.githubusercontent.com/20927667/190245586-2749354c-b06b-4fd8-88fe-3c2bcbd9e651.mov

#### Proposed changes
This PR improves the styling by removing the chevron icon and changing the badge's color to on `#91CAF2` hover:

https://user-images.githubusercontent.com/20927667/190245690-e5a8b7af-19b6-4a9d-b3be-a7afc1357c80.mov

#### Testing Instructions
1. Create a new site using the Newsletter flow
2. Choose a paid plan
3. Once you get to the Launchpad screen, you should see the new style for the **Choose a Plan** task. It should have no chevron icon and the badge background color should be `#91CAF2` on hover

#### Pre-merge Checklist






Related to [#67667](https://github.com/Automattic/wp-calypso/issues/67667)
